src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME_POSITION = 0;
+    private static final int NUMBER_POSITION = 1;
+    private static final String SEPARATOR = ",";
+    private static final String SUPPLY = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] data = readFile(fromFileName);
+        String report = generateReport(data);
```

Comment:
```suggestion
        String data = readFile(fromFileName);
        String report = generateReport(data);
```
readFile should what it name says - read data from file

Reply:
It breaks the logic of the app in the way I can't fix it. And I don't get why I can't use int[] data here.